worldedit-core/src/main/java/com/sk89q/worldedit/world/World.java

```diff
@@ -215,8 +216,9 @@ default int getBlockLightLevel(BlockVector3 position) {
      * Simulate a block being mined at the given position.
      *
      * @param position the position
+     * @return Block default drops
      */
-    void simulateBlockMine(BlockVector3 position);
+    Collection<BaseItemStack> simulateBlockMine(BlockVector3 position);
```

Comment:
Missing FAWE Start/end comments. It needs to specify how it differs from upstream.

Reply:
This would be an API breaking change, causing incompatibility with WE. You need to keep the old method and introduce a new one. It also needs to be a `default` method then to be fully compatible with any classes implementing `World`. (e.g. just call the void method and return an empty list/set)